src/index.spec.ts

```diff
@@ -4,10 +4,12 @@ import { join } from 'path'
 import semver = require('semver')
 import ts = require('typescript')
 import proxyquire = require('proxyquire')
-import { register, create, VERSION } from './index'
+import * as tsNodeTypes from './index'
```

Comment:
Type-only import, since we want to require() the locally-installed ts-node later.

Reply:
`import type` possibly?